### PR TITLE
PR test

### PR DESCRIPTION
**Pull request by hornetmj**
This is first PR.

### CODE REVIEW

**Comment by applys76** [comment]
Confirmed

**Comment by applys76** [comment]
> Confirmed

what is quote reply

**Review by applys76 — APPROVED** [review]

**Review by applys76 — APPROVED** [review]
2nd hot fix approved

File: ctcl/ctcl.c [hunk 1]
```diff
@@ -22,6 +22,8 @@
  *
  */
 
+// This is second commit for hot fix in this PR
```
Comment (applys76) [inline ctcl/ctcl.c:25]: - aaaa

**Comment by hornetmj** [comment]
my PR is perpect!! ^0^

**Pull request closed without merging**